python/pyspark/sql/connect/client.py

```diff
@@ -60,7 +108,18 @@ class ChannelBuilder:
 
     DEFAULT_PORT = 15002
 
-    def __init__(self, url: str) -> None:
+    def __init__(self, url: str, channelOptions: Optional[List[Tuple[str, Any]]] = None) -> None:
+        """
+        Constructs a new channel builder. This is used to create the proper GRPC channel from
+        the connection string.
+
+        Parameters
+        ----------
+        url : str
+            Spark Connect connection string
+        channelOptions: Optional[List[tuple]]
```

Comment:
```suggestion
        channelOptions : list of tuple, optional
```

I referred to https://numpydoc.readthedocs.io/en/latest/format.html#parameters